docker/Dockerfile.oldev

```diff
@@ -3,7 +3,7 @@ FROM openlibrary/olbase:latest
 WORKDIR /openlibrary
 
 USER root
-# oldev currently runs coverstore in the same container:
+# oldev runs coverstore using the same image
```

Comment:
```suggestion
# oldev runs coverstore using the same Docker image
```
Is that right?

Reply:
Yep, talking about the docker image. The old version was no longer correct because they run in separate containers now, but the containers use the same image.